src/components/header.tsx

```diff
@@ -1,18 +1,51 @@
 import type { NextPage } from 'next';
+import Link from 'next/link'
+
 
 const Header: NextPage = () => {
-    const style = {
-        backgroundColor: 'blue',
-        color: 'white',
-        width: '100%',
-        height: '50px'
-    }
     return (
-        <div className='header' style={style}>
-            <div style={{ margin: '10px' }}>
-                CS: 498 - SAP
+        <nav className="sticky p-6 mx-auto bg-white w-full">
+            {/* <!-- Flex Container --> */}
+            <div className="flex items-center justify-between">
+              {/* <!-- Logo --> */}
+              <div className="space-x-6">
+                <Link href="/">
+                  <a className="text-2xl hover:text-dominant">CatSafe</a>
+                </Link>
+              </div>
+
+              {/* <!-- Menu Items --> */}
+              <div className="items-center hidden space-x-4 text-xl md:flex">
+                <Link href="#"><a className="hover:text-dominant">How It Works</a></Link>
+                <Link href="#"><a className="hover:text-dominant">Resources</a></Link>
+                <Link href="#"><a className="hover:text-dominant">About Us</a></Link>
+                <Link href="#"><a className="hover:text-dominant">Contact</a></Link>
+              </div>
+
+              {/* <!-- Button --> */}
+              <div className="items-center hidden text-xl md:flex">
+                <Link href="/login"><a className="py-1 mr-4 hover:text-dominant hover:text-bold">Login</a></Link>
+                <Link href="/signup"><a className="hidden px-6 py-2 text-white rounded bg-accent hover:bg-dominant md:block">Sign Up</a></Link>
+              </div>
+
+              {/* <!-- Hamburger Icon --> */}
+              <button className="block space-y-1.5 md:hidden focus:outline-none">
```

Comment:
Great that you added this but it looks missing when I tested it. Probably because there is no content yet but could you add this afterwards to make our website more reactive?

Reply:
Pushing a commit that has the responsive implemented.